lib/linter.js

```diff
@@ -376,6 +376,16 @@ class Linter {
     return results;
   }
 
+  getConfigForFile(options) {
+    let fileConfig = Object.assign({}, this._moduleStatusCache.getConfigForFile(options));
+
+    delete fileConfig.loadedConfigurations;
+    delete fileConfig._processed;
+    delete fileConfig.shouldIgnore;
```

Comment:
Not sure it matters much, but instead of deleting you could use some "magic"

```suggestion
    let fileConfig = {
      ...this._moduleStatusCache.getConfigForFile(options),
      loadedConfigurations: undefined,
      _processed: undefined,
      shouldIgnore: undefined
    };
```

Since they are `undefined`, when you use `JSON.stringify` later they are completely stripped :tada:

---

feel free to disregard though 😝

Reply:
Ya I like. Accepting.